src/Admin/OptionsMenu.php

```diff
@@ -0,0 +1,483 @@
+								<path d="M21.5767 102.473C16.7617 104.206 6.4336 107.823 3.64062 108.426" stroke="#2459E7" stroke-width="2" stroke-linecap="round"/>
+								<circle cx="3.64091" cy="130.946" r="3.01225" fill="#2459E7"/>
+							</svg>
+
+							</div>
+
+							<div class="settings-welcome__body">
+								<h2>
+									<?php if ( $plugin_configured ) : ?>
+										<?php esc_html_e( 'AMP Settings Configured', 'amp' ); ?>
+
+										<svg width="25" height="25" viewBox="0 0 25 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+											<mask id="check-circle-mask" mask-type="alpha" maskUnits="userSpaceOnUse" x="2" y="2" width="21" height="21">
+												<path fill-rule="evenodd" clip-rule="evenodd" d="M12.7537 2.60938C7.23366 2.60938 2.75366 7.08938 2.75366 12.6094C2.75366 18.1294 7.23366 22.6094 12.7537 22.6094C18.2737 22.6094 22.7537 18.1294 22.7537 12.6094C22.7537 7.08938 18.2737 2.60938 12.7537 2.60938ZM12.7537 20.6094C8.34366 20.6094 4.75366 17.0194 4.75366 12.6094C4.75366 8.19938 8.34366 4.60938 12.7537 4.60938C17.1637 4.60938 20.7537 8.19938 20.7537 12.6094C20.7537 17.0194 17.1637 20.6094 12.7537 20.6094ZM10.7537 14.7794L17.3437 8.18937L18.7537 9.60938L10.7537 17.6094L6.75366 13.6094L8.16366 12.1994L10.7537 14.7794Z" fill="white"/>
+											</mask>
+											<g mask="url(#check-circle-mask)">
+												<rect x="0.753662" y="0.609375" width="24" height="24" fill="#2459E7"/>
+											</g>
+										</svg>
+
+									<?php else : ?>
+										<?php esc_html_e( 'Configure AMP', 'amp' ); ?>
+
+									<?php endif; ?>
+
+								</h2>
+								<p>
+									<?php esc_html_e( 'The AMP configuration wizard can help you choose the best settings for your theme, plugins, and technical capabilities.', 'amp' ); ?>
+								</p>
+
+								<a class="components-button is-primary settings-welcome__button" href="<?php menu_page_url( OnboardingWizardSubmenu::SCREEN_ID ); ?>">
+									<?php if ( $plugin_configured ) : ?>
+										<?php esc_html_e( 'Reopen Wizard', 'amp' ); ?>
+
+									<?php else : ?>
+										<?php esc_html_e( 'Open Wizard', 'amp' ); ?>
+
+									<?php endif; ?>
+									<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
+										<path d="M43.16 10.18c-0.881-0.881-2.322-0.881-3.203 0s-0.881 2.322 0 3.203l16.335 16.335h-54.051c-1.281 0-2.242 1.041-2.242 2.242 0 1.281 0.961 2.322 2.242 2.322h54.051l-16.415 16.335c-0.881 0.881-0.881 2.322 0 3.203s2.322 0.881 3.203 0l20.259-20.259c0.881-0.881 0.881-2.322 0-3.203l-20.179-20.179z" />
+									</svg>
+								</a>
+							</div>
+						</div>
+					</div>
+					<div id="amp-settings-root"></div>
+					<div id="amp-settings-sections" style="display: none !important;">
+						<?php do_settings_sections( AMP_Options_Manager::OPTION_NAME ); ?>
+					</div>
+				</div>
+			</form>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Supported templates section renderer.
+	 *
+	 * @since 1.0
```

Comment:
This `@since` tag is off, since the code moved. Either remove or add a second entry.